server/services/store/sqlstore/migrationstests/boards_migrator_test.go

```diff
@@ -65,12 +65,16 @@ func (bm *BoardsMigrator) runMattermostMigrations() error {
 		return err
 	}
 
-	driver, err := bm.getDriver("")
+	driver, err := bm.getDriver()
 	if err != nil {
 		return err
 	}
 
-	engine, err := morph.New(context.Background(), driver, src)
+	options := []morph.EngineOption{
+		morph.SetStatementTimeoutInSeconds(1000000),
```

Comment:
Statement timeout is 11 days?

Reply:
Yeah, the default value we set was 600 seconds, which is quite small for big DBs. I think we can maybe add a constant to be a year practically being infinite.